### PR TITLE
packit: Land a new upstream release automatically to centos-stream c9s

### PR DESCRIPTION
**Pull request by KKoukiou**
This packit job only makes sure the changes happen in dist-git - no builds.

Relevant documentation: https://packit.dev/docs/configuration/upstream/propose_downstream

Cherry-picked from master PR: https://github.com/rhinstaller/anaconda/pull/5564

### CODE REVIEW

**Review by M4rtinK — APPROVED** [review]
Looks good to me. :)

File: .packit.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    - ./autogen.sh
+    - ./configure
+  create-archive:
+    - "make release"
+    - 'bash -c "ls -1 anaconda-*.tar.bz2"'
+
+jobs:
+
+{% if distro_release == "rawhide" %}
+
```
Comment (jkonecny12) [inline .packit.yml.j2:44]: It would be great to reduce number of blank lines.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me otherwise.

**Pull request merged**